packages/components/src/input-control/label.tsx

```diff
@@ -33,3 +35,15 @@ export default function Label( {
 		</LabelWrapper>
 	);
 }
+
+export function InnerLabel( {
```

Comment:
I noticed that, when the label position is set to `inside` but the `label` prop is still empty, the inner label component still renders an empty `label` element, which also causes the `select` element to shift slightly to the right.

I wonder if we should avoid rendering a `label` at all when its contents are empty?